train_configs/debug_model.toml

```diff
@@ -43,13 +43,14 @@ dataset = "c4_mini"  # supported datasets: c4_mini (45K), c4 (177M)
 pipeline_parallel_degree = 1
 
 [checkpoint]
-enable_checkpoint = false
+enable_checkpoint = true
 folder = "checkpoint"
 interval_type = "steps"
 interval = 5
 model_weights_only = false
 export_dtype = "float32"
 async_mode = "disabled"  # ["disabled", "async", "async_with_pinned_mem"]
+keep_latest_k = 10
```

Comment:
I think we can remove these changes before landing to keep it so that the base `debug_model.toml` does not save checkpoints?

Reply:
Yes sir!